### PR TITLE
ECCV tpms dump

### PR DESCRIPTION
**Pull request by mohituniyal**
Scripts to:
1. Dump reviewer, ac details to csv files
2. Download all pdfs

### CODE REVIEW

File: venues/thecvf.com/ECCV/2020/Conference/python/tpms-get-pdfs.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+submission_invitation = 'thecvf.com/ECCV/2020/Conference/-/Submission'
+
+submissions = openreview.tools.iterget_notes(client, invitation=submission_invitation, details='original')
+
+if not os.path.exists('eccv2020_pdfs'):
+	os.makedirs('eccv2020_pdfs')
+
+for submission in submissions:
+	if 'pdf' in submission.details['original']['content']:
```
Comment (melisabok) [inline venues/thecvf.com/ECCV/2020/Conference/python/tpms-get-pdfs.py]: I think we will run this on the  original notes directly

File: venues/thecvf.com/ECCV/2020/Conference/python/tpms-get-pdfs.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print ('connecting to {0}'.format(client.baseurl))
+
+submission_invitation = 'thecvf.com/ECCV/2020/Conference/-/Submission'
```
Comment (melisabok) [inline venues/thecvf.com/ECCV/2020/Conference/python/tpms-get-pdfs.py]: well you are using the original invitation, but there is not details.original, does this script work?

Comment (mohituniyal) [reply venues/thecvf.com/ECCV/2020/Conference/python/tpms-get-pdfs.py]: Made changes to use the original notes.

File: venues/thecvf.com/ECCV/2020/Conference/python/tpms-get-pdfs.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+submissions = openreview.tools.iterget_notes(client, invitation=submission_invitation)
+
+if not os.path.exists('eccv2020_pdfs'):
+	os.makedirs('eccv2020_pdfs')
+
+for submission in submissions:
```
Comment (melisabok) [inline venues/thecvf.com/ECCV/2020/Conference/python/tpms-get-pdfs.py]: can we add tqdm here?

Comment (mohituniyal) [reply venues/thecvf.com/ECCV/2020/Conference/python/tpms-get-pdfs.py]: done.

**Pull request merged**